lib/plaindate.ts

```diff
@@ -1,5 +1,5 @@
 import * as ES from './ecmascript';
-import { GetIntrinsic, MakeIntrinsicClass } from './intrinsicclass';
+import { MakeIntrinsicClass } from './intrinsicclass';
```

Comment:
Where is `MakeIntrinsicClass` used in this and other files where it's imported?

Reply:
Way down at the bottom :smile:

Reply:
Oh whoops, I saw the green and thought "new line" but it's just removing GetIntrinsic; the `MakeIntrinsicClass` was already there.